### DISCUSSION TITLE
markup for command line output

### DISCUSSION THREAD

**Discussion by fanbyprinciple** [github-linguist/linguist#6100] [Q&A]
Hello Everyone !

Please may i know if there is a codeblock for command line output both from windows and unix based oses.

**Comment by Alhadis** [comment]
I take it that you're looking for something like this...?

~~~console
$ echo "Hello, world"
Hello, world
~~~

That's the extent of the output's highlighting though, and I'm not sure how well it plays with CMD.EXE prompts...

**Reply by fanbyprinciple** [reply]
yes, exactly that for cmd.exe prompts. thank you!
